### ISSUE TITLE
int[] field data compare - TypeError: value.replace is not a function

### ISSUE THREAD

**Issue by TomKohrah** [michaelsogos/pg-diff#30]
Hi @michaelsogos,
thank you for this awesome tool! Sadly, I'm experiencing a TypeError with data-compare as soon as a table contains a filled integer[] field. If data-compare is disabled or the field contains null, everything works fine. I'm looking forward for any help. Thank you!

This is how it looks like:
```
$ pg-diff -c development initial-script
                              _   _    __    __                  _   _
  _ __     __ _            __| | (_)  / _|  / _|           ___  | | (_)
 | '_ \   / _` |  _____   / _` | | | | |_  | |_   _____   / __| | | | |
 | |_) | | (_| | |_____| | (_| | | | |  _| |  _| |_____| | (__  | | | |
 | .__/   \__, |          \__,_| |_| |_|   |_|            \___| |_| |_|
 |_|      |___/

     Author: [object Object]
    Version: 2.0.4
 PostgreSQL: 9.6+
    License: MIT
Description: PostgreSQL schema and data comparing tool


CONFIGURED OPTIONS
         Script Author: pg-diff-cli
      Output Directory: /data/pg-diff/sqlscripts
     Schema Namespaces: public
          Data Compare: ENABLED

[||||||||||||||-----] 75% - SEQUENCE objects have been compared
TypeError: value.replace is not a function
    at Object.__generateSqlFormattedValue (/usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/sqlScriptGenerator.js:598:22)
    at Object.generateInsertTableRecordScript (/usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/sqlScriptGenerator.js:545:26)
    at /usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/api/CompareApi.js:1180:10
    at Array.forEach (<anonymous>)
    at Function.compareTableRecords (/usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/api/CompareApi.js:1140:37)
    at Function.compareTablesRecords (/usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/api/CompareApi.js:1018:31)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Function.compare (/usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/api/CompareApi.js:63:9)
    at async PgDiff.compare (/usr/local/lib/node_modules/pg-diff-cli/node_modules/pg-diff-api/src/index.js:36:10)
    at async Run (/usr/local/lib/node_modules/pg-diff-cli/main.js:79:26)
```

Steps to reproduce:
- Set up two postgres databases version 11.9
- Create table in sourceClient database using attached diff_test.sql, second db remains empty
- Enable data comparison and run `pg-diff -c development initial-script`

pg-diff-config.json:
```
...
            "dataCompare": {
                "enable": true,
                "tables": [
                    {
                        "tableName": "diff_test",
                        "tableSchema": "public",
                        "tableKeyFields": ["id"]
                    }
                ]
            }
...
```

diff_test.sql
```
create table public.diff_test (
    id   serial not null
        constraint diff_test_pk
            primary key,
    nums integer[]
);
INSERT INTO public.diff_test (id, nums) VALUES (1, '{1,2}');
```

**Comment by michaelsogos** [comment]
Dear @TomKohrah ,

I don't have file to reproduce the issue.
Can you share a link with any Cloud Drive (like wetransfer or dropbox or onedrive)?

**Comment by michaelsogos** [comment]
Dear @TomKohrah 

I fixed it, please download v2.0.5, or if you are using pg-diff-api then to v1.0.9
